coffeecard/CoffeeCard.Library/Services/PurchaseService.cs

```diff
@@ -22,7 +23,7 @@ public PurchaseService(CoffeeCardContext context)
             _context = context;
         }
 
-        public Purchase RedeemVoucher(string voucherCode, IEnumerable<Claim> claims)
+        public async Task<Purchase> RedeemVoucher(string voucherCode, IEnumerable<Claim> claims)
```

Comment:
Should be named Async

Reply:
I could do it but this is also just the v1 one method which we have for one month or so :)